packages/palette/src/elements/Marquee/Marquee.tsx

```diff
@@ -55,19 +62,22 @@ const Item = styled.div`
 export interface MarqueeProps extends BoxProps {
   variant?: keyof typeof VARIANTS
   speed?: string
+  mobileSpeed?: string
   color?: string
   marqueeText: string
+  textSize?: TextVariant
```

Comment:
If we're using `TextVariant` I'd name this `textVariant`

Reply:
👍